src/mutation.cpp

```diff
@@ -407,12 +407,7 @@ void player::activate_mutation( const trait_id &mut )
     if( mut == trait_WEB_WEAVER ) {
         g->m.add_field(pos(), fd_web, 1, 0);
         add_msg_if_player(_("You start spinning web with your spinnerets!"));
-    } else if( mut == "WEB_ROPE" ) {
-        add_msg_if_player(_("You spin a rope from your silk."));
-        item rope( "rope_30" );
-        i_add_or_drop( rope );
-        tdata.powered = false;
-    } else if (mut == "BURROW"){
+    }  else if (mut == "BURROW"){
```

Comment:
Excess space before `else`. Also, should be astyled as `if( mut == "BURROW" ) {`.